### PR TITLE
Show appointment edits as activities

### PR DESCRIPTION
**Pull request by benlovell**
Converts and displays appointment edits (provided by `audited`) as
activities in the feed.

I went with an STI approach to make the addition of further activity types
easier in future. Having the activity `message` as simple text affords us
lots of possibilities / control over how and what we display as the
activity summary.

The activities themselves are linked to the booking request and an owning
user. For those activities that do not have a natural owner, we display
'Somebody' as a placeholder.

### CODE REVIEW

**Comment by benlovell** [comment]
Oh yeah, pretty sure my javascript sucks and the tests will fail in travis.

**Comment by Guntrisoft** [comment]
@benlovell @benbarnett made some adjustments to the toggling functionality 8cbccfc2aaf4a01cc0a3be6a148ee2a069da540f

File: app/views/appointments/edit.html.erb [hunk 1]
```diff
@@ -17,27 +17,22 @@
...
-              </td>
-            </tr>
-          <% end %>
-        </table>
+        </ul>
+        <% if @activities.count > 1 %>
+          <ul class="activity-feed__list t-hidden-activities" id="collapse">
+            <%= render(@activities - Array(@activities.first)) %>
+          </ul>
+        <% end %>
```
Comment (benlovell) [inline app/views/appointments/edit.html.erb]: This `end` and the `if` below could be removed.

**Comment by Guntrisoft** [comment]
@benbarnett @benlovell I'm happy with the rest of this if you guys are

**Pull request merged**